lib/daru/core/group_by.rb

```diff
@@ -293,6 +306,17 @@ def apply_method_index
         end
       end
 
+      def resultant_context(multi_index_tuples, names)
```

Comment:
Since input dataframe is called as `context ` in [this line](https://github.com/SciRuby/daru/pull/330/files#diff-e2feb9bd3b453b5a4adb14512ac7e8ebL22), so I thought I can create new dataframe using method named `resultant_context `.